src/main/java/com/targomo/client/api/quality/criterion/PoiCriterionDefinition.java

```diff
@@ -4,7 +4,7 @@
 
 import java.util.Set;
 
-public interface PoiCriterionDefinition {
+public interface PoiCriterionDefinition extends CriterionDefinitionInterface{
 
     CriterionType getType();
```

Comment:
Do we still need this since it is defined in CriterionDefinitionInterface? Not sure if that can cause any failure related to in some of our annotated classes.

Reply:
you're right I removed it